docs/source/gcloud_topo.rst

```diff
@@ -54,5 +54,6 @@ Google Cloud Topologies
 
 .. note::
 
-   
+  Source of the above mentioned examples can be fount at following link: 
```

Comment:
The repercussions of using a vi editor without a spell check.
fixing it with followup commit .